etc/systemd/system/50-zfs.preset.in

```diff
@@ -5,4 +5,5 @@ enable zfs-import.target
 enable zfs-mount.service
 enable zfs-share.service
 enable zfs-zed.service
+enable zfs-volume-wait.service
```

Comment:
I thought we determined this service should be disabled by default for now.

Reply:
Since `zfs-volume.target` is disabled by default, this service will not be started by anyone even though it is enabled. It will be started only if `zfs-volume.target` is `WantedBy` or `RequiredBy` another service.  I'm not sure if that's a common way to do those things in systemd, but with this configuration `zfs-volume-wait.service` would not start on boot by default.

Reply:
I see, that makes sense.  I couldn't find documentation for the established convention either and that seems reasonable.